### DISCUSSION TITLE
Not able to Login in my wiki

### DISCUSSION THREAD

**Discussion by rumi-2** [requarks/wiki#5428] [Help / Questions]
System configuration : Windows 10

After installation from following link (https://docs.requarks.io/install/windows), I used Windows Powershell to launch my wiki using command "node server" and I was able to access the wiki without any authentication.
Recently, I cleared my chrome history, cache, all site passwords and settings. Now when I launch the wiki using Powershell I am asked to login. I used my gmail account to login but am getting following warning while launching the wiki: 
"warn: Cannot send email because mail is not setup in the administration area!" (Screenshot attached below )
![git](https://user-images.githubusercontent.com/22032161/175805837-718ddcd2-7fe4-4cf0-8727-efeede2c8692.PNG)

Upon checking it, wiki.js site (https://docs.requarks.io/auth/local) suggests to add the gmail using Administrator area in WIKI but problem is I am **not able to LOGIN** at all, thus tab is not accessible.

Is there a way to **login in wiki.js without using email** and then I can add the gmail account to authentication.  Kindly, help this is URGENT @NGPixel 

Update:
Currently I am trying to use github to authenticate (https://docs-legacy.requarks.io/wiki/install/authentication) but unable to find auth section in config.yml. Will this approach work and where can I find such section in yml?

**Comment by NGPixel** [comment]
Use the root account (the email and password you entered while installing Wiki.js the first time). If you forgot it, you can reset it via the DB:
https://docs.requarks.io/en/troubleshooting#how-to-manually-reset-the-admin-password

You can't reset your password via the UI because you didn't setup the mail settings under Administration > Mail.

---

The last link you provided is for Wiki.js 1.x, NOT 2.x. All authentication settings are managed via the UI, not config.yml.

**Reply by rumi-2** [reply]
I have used pgAdmin tool to reset password and verified the same in the table but still am getting the same error while logging in via UI. Any idea on where I can look for troubleshooting?
